### ISSUE TITLE
[seek help]My custom dataset has 90 categories and 1300 images in which half number is nature images,the rest specimen.The dataset is imbalanced among classes. I havn't augmented each image 10 times to create 13000 images. I split train and val set with ratio 8:2. I had a try to employ retinanet with resnet(18,50,101)and ssd300.But overfitting appears  so that train set performs well but validation set is bad . How can I do my experiment? Which model can I try? Please help me if you have solved similar situation ,Thanks!

### ISSUE THREAD

**Issue by lyy-fighting** [open-mmlab/mmdetection#2872]

**Comment by ZwwWayne** [comment]
You could try to finetune a model pre-trained on COCO following the [tutorial](https://github.com/open-mmlab/mmdetection/blob/master/docs/tutorials/finetune.md)

**Comment by hellock** [comment]
Use a short title please.
